great_ape_safe/ape_api/ebtc.py

```diff
@@ -508,6 +530,105 @@ def schedule_or_execute_timelock(self, timelock, target, data, salt):
                 timelock, target.address, 0, data, EmptyBytes32, salt, delay + 1
             )
 
+    def schedule_batch_timelock(self, timelock, targets, values, data, salt, delay):
```

Comment:
revising the `delay` param something i missed on prev reviews on any method re scheduling a timelock tx is that the `delay` value does not have some validation process before proceeding. maybe worth having a strong assert against `delay >= MIN_DELAY` or something along those lines to flag quickly

Reply:
Good call, added.